Cargo.toml

```diff
@@ -31,7 +31,7 @@ async-trait = { version = "0.1", optional = true }
 rocksdb = { version = "0.14", default-features = false, features = ["snappy"], optional = true }
 cc = { version = ">=1.0.64", optional = true }
 socks = { version = "0.3", optional = true }
-hwi = { version = "0.5", optional = true, features = ["use-miniscript"] }
+hwi = { version = "0.5", optional = true, features = [ "use-miniscript"] }
```

Comment:
Since I used the git revert command it undid this spacing fix too. To keep it simple I'd rather not make any other changes in this PR but to revert the original commit. This can be fixed next time we touch the Cargo.toml file.